attention_system_base:
  ros__parameters:
    use_sim_time: true
    pid_min_ref: 5.0
    pid_max_ref: 90.0
    pid_min_output: 0.0
    pid_max_output: 0.2
    